api/v1alpha/api.proto

```diff
@@ -485,3 +503,63 @@ service PublicAPI {
         // the stream.
         rpc GetLogs(GetLogsRequest) returns (stream GetLogsResponse) {}
 }
+
+// FetchImagesRequest defines the request for FetchImages.
+message FetchImagesRequest {
+        // Type of the image, e.g. aci, docker, oci, etc.
+        Image type = 1;
+
+        // Resolvable names of the image, used for discovery, can be string, hash, URL, file path.
+        repeated string names = 2;
```

Comment:
@yifan-gu with the distribution you only need one fields (distribution string). The reason why you don't need the image type are explained in #2953